webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue

```diff
@@ -100,7 +101,7 @@ export default {
       return this.advancedFilterCount > 0 ? `(${this.advancedFilterCount})`:'';
     },
     isMobile() {
-      return this.$vuetify.breakpoint.width < 768;
+      return this.$vuetify.breakpoint.name === 'sm';
```

Comment:
```suggestion
      return this.$vuetify.breakpoint.width < 768;
```
nope, the xs breakpoint is a Mobile breakpoint too, please use CSS classes instead to display or hide elements with `d-none d-md-flex` by example.